DataFormats/Common/interface/EventInfo.h

```diff
@@ -0,0 +1,15 @@
+#ifndef DataFormats_Common_EventInfo_h
```

Comment:
I'm not keen on the name EventInfo since it doesn't explain its purpose. How about ```EventFromFileInfo```